### PR TITLE
build: adds manage vault side drawer

### PR DESCRIPTION
**Pull request by matthew-garrett**
- adds manage vault side drawer
- addresses previous PR comments

### CODE REVIEW

File: packages/features/portfolio/src/side-drawers/manage-vault/manage-vault.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+              {label}
+            </H4>
+          </ExternalLink>
+        </WalletButton>
+      ))}
+    </Box>
+  );
+};
+
+const Title = styled(LabelValue)(
```
Comment (jeffywu) [inline packages/features/portfolio/src/side-drawers/manage-vault/manage-vault.tsx:94]: Should this be "SideNavText" instead?

File: packages/features/portfolio/src/side-drawers/manage-vault/manage-vault.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  ({ theme }) => `
+  margin-bottom: ${theme.spacing(2.5)};
+  margin-top: ${theme.spacing(5)};
+  color: ${theme.palette.borders.accentDefault};
+  text-transform: uppercase;
+  `
+);
+
+const WalletButton = styled(Box)(
```
Comment (jeffywu) [inline packages/features/portfolio/src/side-drawers/manage-vault/manage-vault.tsx:103]: Maybe rename this?

File: packages/features/portfolio/src/side-drawers/manage-vault/use-manage-vault.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { FormattedMessage } from 'react-intl';
+
+export function useManageVault() {
+  const reduceLeverageOptions = [
+    {
+      label: <FormattedMessage defaultMessage={'Deposit Collateral'} />,
+      href: '',
+    },
+    {
+      label: <FormattedMessage defaultMessage={'Withdraw and repay debt'} />,
```
Comment (jeffywu) [inline packages/features/portfolio/src/side-drawers/manage-vault/use-manage-vault.tsx:11]: Capitalize "Repay Debt"

File: packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx [hunk 4]
```diff
@@ -190,11 +191,10 @@ const WalletButton = styled('div', {
   `
 );
 
-const Title = styled(Typography)(
+const Title = styled(LabelValue)(
```
Comment (jeffywu) [inline packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx:194]: Maybe we can also make this "SideNavText" as well

Comment (matthew-garrett) [reply packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx:194]: SideNavText is not the correct font size according to the mocks

Comment (matthew-garrett) [reply packages/shared/wallet/src/settings-side-drawer/settings-side-drawer.tsx:194]: Which is why I went with LabelValue

**Pull request closed without merging**